scipy/signal/filter_design.py

```diff
@@ -2071,8 +2071,10 @@ def iirdesign(wp, ws, gpass, gstop, analog=False, ftype='ellip', output='ba',
 
     Parameters
     ----------
-    wp, ws : float
-        Passband and stopband edge frequencies.
+    wp, ws : float or array like
```

Comment:
I think we need to be more specific here, like `array like, shape (2,)`. Otherwise there is an implication that this might operate on arrays of pass/stopband pairs (e.g., along `axis=-1`).

Maybe there should even be a check below like:
```
if wp.shape != ws.shape or wp.shape not in [(1,), (2,)]:
     raise ValueError('wp and ws must have one or two elements and be '
                      'the same shape, got %s and %s'
                      % (ws.shape, wp.shape))

Reply:
Good idea @larsoner ! That is a bit more specific than the current check